floodrelief/settings.py

```diff
@@ -184,4 +160,4 @@ def get_list(text):
 	os.path.join(BASE_DIR, 'static'),
 )
 
-ADMIN_SITE_HEADER = "Keralarescue Dashboard"
+ADMIN_SITE_HEADER = "test"
```

Comment:
Do you need this change?

Reply:
I guess this was added when i copied the settings from this repo
https://github.com/vigneshhari/keralarescue_test_settings
Is this not relevant here?